h2o-py/h2o/h2o.py

```diff
@@ -320,7 +323,7 @@ def upload_file(path, destination_frame=None, header=0, sep=None, col_names=None
     return H2OFrame()._upload_parse(path, destination_frame, header, sep, col_names, col_types, na_strings)
 
 
-def import_file(path=None, destination_frame=None, parse=True, header=0, sep=None, col_names=None, col_types=None,
+def import_file(path=None, pattern="", destination_frame=None, parse=True, header=0, sep=None, col_names=None, col_types=None,
```

Comment:
It would be better use `None` to express no-pattern specified. The `""` value still represent a pattern matching all. It is just about message which we would like to send to users.